sqlmodel/sql/expression.py

```diff
@@ -24,36 +22,15 @@
 
 _TSelect = TypeVar("_TSelect")
 
-# Workaround Generics incompatibility in Python 3.6
-# Ref: https://github.com/python/typing/issues/449#issuecomment-316061322
-if sys.version_info.minor >= 7:
+class Select(_Select[_TSelect], Generic[_TSelect]):
+    inherit_cache = True
 
-    class Select(_Select, Generic[_TSelect]):
-        inherit_cache = True
-
-    # This is not comparable to sqlalchemy.sql.selectable.ScalarSelect, that has a different
-    # purpose. This is the same as a normal SQLAlchemy Select class where there's only one
-    # entity, so the result will be converted to a scalar by default. This way writing
-    # for loops on the results will feel natural.
-    class SelectOfScalar(_Select, Generic[_TSelect]):
-        inherit_cache = True
-
-else:
-    from typing import GenericMeta  # type: ignore
-
-    class GenericSelectMeta(GenericMeta, _Select.__class__):  # type: ignore
-        pass
-
-    class _Py36Select(_Select, Generic[_TSelect], metaclass=GenericSelectMeta):
-        inherit_cache = True
-
-    class _Py36SelectOfScalar(_Select, Generic[_TSelect], metaclass=GenericSelectMeta):
-        inherit_cache = True
-
-    # Cast them for editors to work correctly, from several tricks tried, this works
-    # for both VS Code and PyCharm
-    Select = cast("Select", _Py36Select)  # type: ignore
-    SelectOfScalar = cast("SelectOfScalar", _Py36SelectOfScalar)  # type: ignore
+# This is not comparable to sqlalchemy.sql.selectable.ScalarSelect, that has a different
+# purpose. This is the same as a normal SQLAlchemy Select class where there's only one
+# entity, so the result will be converted to a scalar by default. This way writing
+# for loops on the results will feel natural.
+class SelectOfScalar(_Select[_TSelect], Generic[_TSelect]):
+    inherit_cache = True
```

Comment:
You are probably right, but this is copied from the original code as is 

https://github.com/tiangolo/sqlmodel/pull/563/files/050cf02a35b670eedeb814606d27869c3a723328#diff-976d5f3bdb62b7724d2e776bb785b98e13d96aac3a72d117b047467828756851L34-L39

Reply:
Roger. However, I think it's better to put the description in the class docstring. That way, it will be parsed/handled by whatever instrument usually does this (for example., your favourite IDE + some IntelliSense).

Reply:
Until there is a valid reason from the original authors, of course.